src/AdvanceSteelNodes/base/ASProperty.cs

```diff
@@ -94,6 +115,29 @@ public void UpdateASObject(object asObjectToUpdate)
       }
     }
 
+    public bool GetValueFromSteelDBObject(SteelDbObject steelObject)
```

Comment:
Only a suggestion ... I believe that UpdateValueFromSteelDBObject is is a better name candidate, as it does not return any value